app/views/settings/kubelet_compute_resources_reservations/index.html.slim

```diff
@@ -0,0 +1,143 @@
+= render 'settings/apply'
+
+.alert.alert-danger role='alert'
+  p Warning: Entering invalid values for any of the following settings will cause
+    the nodes to enter into a broken state.
+
+h2 Compute resources reservations
+
+p Every node of the Kubernetes cluster has a kubelet instance running. By default,
+  the kubelet process will try to use all available resources on each node. This
+  behaviour can lead to resource starvation for critical system services as well
+  as for Kubernetes' own components.
+
+p To prevent this, it is possible to instruct kubelet to reserve a certain
+  amount of resources for the host system and for Kubernetes core services on each node.
+  The Kubernetes scheduler takes these limits into account by when deciding
+  on which node to schedule a certain pod.
+
+= form_for :kubelet_compute_resources_reservations, url: settings_kubelet_compute_resources_reservations_path, method: :post do |f|
+  .panel.panel-default
```

Comment:
It's clear that we have 3 group of fields, so it would be nice to extract them to its own partials.

Reply:
I guess you forgot to extract the panels to its own partials? But this file is still fine since it's under 100 lines.